### ISSUE TITLE
Issues with conda environment

### ISSUE THREAD

**Issue by aburgasser** [pyspextool/pyspextool#161]
I installed the most recent version of pyspextool following the instructions on README, in particular creating a conda environment. 

**PROBLEM 1:** If I'm in my regular zsh environment, python/ipython/jupyter cannot find pyspextool despite the folder being in the sys.path:

```
(base) adam@onaeao ~ % ipython
Python 3.12.4 | packaged by Anaconda, Inc. | (main, Jun 18 2024, 10:07:17) [Clang 14.0.6 ]
Type 'copyright', 'credits' or 'license' for more information
IPython 8.25.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import pyspextool
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 import pyspextool

ModuleNotFoundError: No module named 'pyspextool'

In [2]: import sys

In [3]: sys.path
Out[3]: 
[ ...
 '/Users/adam/python_codes/pyspextool',
...]
```
**PROBLEM 2**: if I move to the conda environment I've set up, my system cannot find ipython/jupyter (but can find python - but a different version

```
(base) adam@onaeao ~ % conda activate pyspextool_3.11
(pyspextool_3.11) adam@onaeao ~ % ipython
zsh: command not found: ipython
(pyspextool_3.11) adam@onaeao ~ % jupyter notebook
zsh: command not found: jupyter
(pyspextool_3.11) adam@onaeao ~ % python
Python 3.11.9 | packaged by conda-forge | (main, Apr 19 2024, 18:34:54) [Clang 16.0.6 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pyspextool
Matplotlib is building the font cache; this may take a moment.
>>> pyspextool.
pyspextool.combine            pyspextool.pyspextool_setup(
pyspextool.config             pyspextool.pyspextoolerror
pyspextool.extract            pyspextool.setup_utils
pyspextool.fit                pyspextool.telluric
pyspextool.io                 pyspextool.utils
pyspextool.plot               
```

Does this mean I need to reinstall all of the other packages that are already installed in my base setup? These clearly needs fixing/more instructions.

If I run `pip install -e ./ `without making the conda environment, everything seems to run fine, but of course this might mean that some packages are in different versions.

**Comment by aburgasser** [comment]
2 options:
- make jupyter notebook (but not ipython) an optional dependency in environment.yaml
- remind user they will need to install ipython, jupyter notebook, etc. in new environment

**Comment by kelle** [comment]
I've implemented option 2 in #167.
